xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp

```diff
@@ -201,7 +202,16 @@ enum AVPixelFormat CDVDVideoCodecDRMPRIME::GetFormat(struct AVCodecContext* avct
     }
   }
 
-  CLog::Log(LOGERROR, "CDVDVideoCodecDRMPRIME::{} - unsupported pixel format", __FUNCTION__);
+  std::string formats;
+  for (int n = 0; fmt[n] != AV_PIX_FMT_NONE; n++)
+  {
+    if (n > 0)
+      formats += ", ";
```

Comment:
Instead of manual formatting can we use our helpers.

Pseudo code (done on mobile)
```
std::vector<std::string> formats

Then in loop:

formats.emplace_back(av_get_pix_fmt_name(fmt[n]));

Then use StringUtils::Join when logging.
```

Reply:
Updated to use `Join`.